config/database.yml

```diff
@@ -23,7 +23,8 @@ development:
 
 test:
   <<: *defaults
-  database: <%= ENV.fetch('DATABASE_NAME', 'timeoverflow_test') %>
+  host: localhost
+  database: 'timeoverflow_test'
```

Comment:
Aren't these the values it was using so far? Or was the ENV var set?

Reply:
Yep, env var is set to `timeoverflow_development` D:

Reply:
:trollface: